src/client.ts

```diff
@@ -367,6 +371,15 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
     return await this.userContext.fetchQualifiedSegments(options);
   }
 
+  public getQualifedSegments(): string[] | null {
```

Comment:
Do we need this extra API? They can get/set qualifiedSegments after getUserContext().

Reply:
During bug bash React Native, I had no way to verify what qualified segments were given.  We're hiding this from React but not to JS. My thought was that we should have a way to view. 

This was part of the gap. 

Perhaps a call is good to review tomorrow?

Reply:
After sleeping on it, I think I understand. 

Are you looking for something like:
`const qualifedSetments = optimizelyClient.getUserContextInstance({id: 'matjaz-user-2'})?.qualifiedSegments;`